templates/etc/icinga2/scripts/twilio-phone-notification.sh.j2

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+account_sid="{{ icinga2_master_twilio_account_sid }}"
+auth_token="{{ icinga2_master_twilio_auth_token }}"
+phone_number="{{ icinga2_master_twilio_phone_from }}"
+application_sid="{{ icinga2_master_twilio_phone_application_sid }}"
+
+PROG="`basename $0`"
+ICINGA2HOST="`hostname`"
+
+## Function helpers
+Usage() {
+cat << EOF
+
+Required parameters:
+  -r USERPHONE (\$user.pager\$)
+EOF
+}
+
+Help() {
+  Usage;
+  exit 0;
+}
+
+Error() {
+  if [ "$1" ]; then
+    echo $1
+  fi
+  Usage;
+  exit 1;
+}
+
+## Main
+while getopts hr: opt
+do
+  case "$opt" in
+    h) Help ;;
+    r) USERPHONE=$OPTARG ;; # required
+   \?) echo "ERROR: Invalid option -$OPTARG" >&2
+       Error ;;
+    :) echo "Missing option argument for -$OPTARG" >&2
+       Error ;;
+    *) echo "Unimplemented option: -$OPTARG" >&2
+       Error ;;
+  esac
+done
+
+shift $((OPTIND - 1))
+
+## Keep formatting in sync with mail-service-notification.sh
+for P in USERPHONE ; do
+	eval "PAR=\$${P}"
+
+	if [ ! "$PAR" ] ; then
+		Error "Required parameter '$P' is missing."
+	fi
+done
+
+## Send the sms using the Twilio API
+echo "sent call to ${USERPHONE} now: $(date -u)" >> ./logfile.log
+
+curl -X POST \
+  https://api.twilio.com/2010-04-01/Accounts/${account_sid}/Calls.json \
```

Comment:
```suggestion
  https://api.twilio.com/{{ icinga2_master__twilio_api_version}}/Accounts/${account_sid}/Calls.json \
```
add version in variable, so we can more easily track changes in their API

Reply:
Let's create an issue for this